packages/react-server/src/__tests__/ReactFlightAsyncDebugInfo-test.js

```diff
@@ -491,7 +491,7 @@ describe('ReactFlightAsyncDebugInfo', () => {
               ],
               "start": 0,
               "value": {
-                "status": "halted",
```

Comment:
Why was this considered halted? We're not halting in this test as far as I can tell.

Reply:
If it's not resolved within a microtask it's considered halted. That's not the case when a debug channel is connected anymore since then we ask for it on demand instead.